subsys/llext/llext.c

```diff
@@ -770,6 +825,14 @@ static int do_llext_load(struct llext_loader *ldr, struct llext *ext,
 	ldr->sect_cnt = ldr->hdr.e_shnum;
 	ext->alloc_size += sect_map_sz;
 
+#ifdef CONFIG_USERSPACE
+	ret = k_mem_domain_init(&ext->mem_domain, 0, NULL);
+	if (ret != 0) {
```

Comment:
you can't initialize a memory domain without any partition AFAIK

Reply:
the docs specifically say its allowed and note NULL may be used with 0 parts https://docs.zephyrproject.org/latest/kernel/usermode/memory_domain.html#c.k_mem_domain_init